resources/views/app/includes/sections/top_nav.blade.php

```diff
@@ -24,6 +24,10 @@
                         &nbsp;
                         {{ Auth::user()->username }}<i class="material-icons right">arrow_drop_down</i></a>
 
+                        @if( Auth::user()->hasChannelNotifications())
+                            <a class="waves-effect waves-light modal-trigger" href="{{ route('notifications') }}"> <span class="badge new">{{ Auth::user()->newChannelsCount() }}</a></span>
```

Comment:
**Code-Quality - Nitpick**[
![screenshot 2016-06-06 15 22 27](https://cloud.githubusercontent.com/assets/9655574/15826067/078337b8-2bfe-11e6-8ee2-fb3081a384c8.png)
@andela-celisha-wigwe Please let's make the UI here look like that above.